Chat Assignment Usage Notes (#352)
hellohaptik/docs_haptik: docs/agent-chat/chat-assignment.md

3. Con(s): Chat load is not balanced across agents i.e., fast agents are the ones who close chats quickly thus making the call to request for chats frequently. This breaks the 10 seconds cycle and creates uneven distribution of chats.

### Note

> 1. If you have a single client and multiple businesses set up, all businesses must be the same algorithm

Reviewer: Isn't multiple businesses not allowed any more?

> 2. If you have multiple client setup, one can ideally have different algorithms for both, provided there are no shared agents between the two clients.

Reviewer: what is multiple client set up? is it multiple partner or multiple business?

> 2. If you have multiple client setup, one can ideally have different algorithms for both, provided there are no shared agents between the two clients.

Reviewer: ```suggestion
2. If there are multiple clients, one can ideally have different algorithms for all, provided there are no shared agents between the teams of different clients.
```
